internal/cmd/options.go

```diff
@@ -15,3 +20,25 @@ func (opts *SignerFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
 	SetPflagKey(fs, &opts.Key)
 	SetPflagSignatureFormat(fs, &opts.SignatureFormat)
 }
+
+// LoggingFlagOpts option struct.
+type LoggingFlagOpts struct {
+	Debug   bool
+	Verbose bool
+}
+
+// ApplyFlags applies flags to a command flag set.
+func (opts *LoggingFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
+	fs.BoolVarP(&opts.Debug, "debug", "d", false, "debug mode")
+	fs.BoolVarP(&opts.Verbose, "verbose", "v", false, "verbose mode")
+}
+
+// SetLoggerLevel sets up the logger based on common options.
+func (opts *LoggingFlagOpts) SetLoggerLevel(ctx context.Context) (context.Context, log.Logger) {
```

Comment:
Looks like log.Logger in the output is never used?

Reply:
Updated.